### PR TITLE
fix: regression on alias generation introduced by column selection

### PR DESCRIPTION
**Pull request by lukapeschke**
depends on #190

### CODE REVIEW

**Comment by lukapeschke** [comment]
@PrettyWood woops just noted this bug was introduced by the column selection implem :grimacing:  Once this and #190 are merged, I'll be able to provide new benchmarks on the eager loading PR  :slightly_smiling_face:

File: src/types/excelsheet.rs [hunk 1]
```diff
@@ -352,17 +355,23 @@ impl ExcelSheet {
 
         let available_columns = sheet.get_available_columns();
 
+        let mut aliased_available_columns = Vec::with_capacity(available_columns.len());
+
+        available_columns.iter().for_each(|column_name| {
+            aliased_available_columns.push(alias_for_name(column_name, &aliased_available_columns))
```
Comment (PrettyWood) [inline src/types/excelsheet.rs:361]: ah right yes!
why not
```rs
let aliased_available_columns = available_columns
    .iter()
    .map(|column_name| alias_for_name(column_name, &available_columns))
    .collect::<Vec<_>>()
``` 
?

**Review by PrettyWood — APPROVED** [review]
If my suggestion is accepted feel free to rebase and push force to have cleaner PR

File: src/types/excelsheet.rs (hunk 1 above)
Comment (PrettyWood) [reply src/types/excelsheet.rs]: [2f1445](https://github.com/ToucanToco/fastexcel/pull/191/commits/42f1445c1c94b241eea7587a037db2a2349d96e9)

**Pull request merged**